deploy/crds/logging.openshift.io_clusterlogforwarders_crd.yaml

```diff
@@ -0,0 +1,203 @@
+                          value 'COPY' means copy from incoming log record.
+                        enum:
+                        - kernel
+                        - user
+                        - mail
+                        - daemon
+                        - auth
+                        - syslog
+                        - lpr
+                        - news
+                        - uucp
+                        - cron
+                        - authpriv
+                        - ftp
+                        - ntp
+                        - security
+                        - console
+                        - cron
+                        - local0
+                        - local1
+                        - local2
+                        - local3
+                        - local4
+                        - local5
+                        - local6
+                        - local7
+                        type: string
+                      payloadKey:
+                        description: PayloadKey specifies record field to use as payload.
+                        type: string
+                      severity:
+                        description: Severity for outgoing syslog records. Special
+                          value 'COPY' means copy from incoming log record.
+                        enum:
+                        - Emergency
+                        - Alert
+                        - Critical
+                        - Error
+                        - Warning
+                        - Notice
+                        - Informational
+                        - Debug
+                        type: string
+                      tagKey:
+                        description: TagKey specifies a record field  to  use as tag.
+                        type: string
+                      trimPrefix:
+                        description: TrimPrefix is a prefix to trim from the tag.
+                        type: string
+                    type: object
+                  type:
+                    description: Type of output plugin, for example 'syslog'
+                    type: string
+                required:
+                - destination
+                - name
+                - type
+                type: object
+              type: array
+            pipelines:
+              description: Pipelines select log messages to send to outputs.
+              items:
+                properties:
+                  inputRefs:
+                    description: InputRefs lists the names of inputs to this pipeline.
+                      By default, all available log streams are forwarded.
+                    items:
+                      enum:
+                      - Application
+                      - Infrastructure
+                      - Audit
+                      type: string
+                    type: array
+                  name:
+                    description: Name of the pipeline. Optional, needed for patch
```

Comment:
Consider making this required.  it makes it easier to identify in status.

Reply:
:+1: also if its needed for patch updates it would make sense to make it required

Reply:
+1